homeassistant/components/roon/media_player.py

```diff
@@ -350,6 +373,11 @@ def media_seek(self, position: float) -> None:
 
     def set_volume_level(self, volume: float) -> None:
         """Send new volume_level to device."""
+
+        if self._attr_volume_fixed:
+            _LOGGER.error("Cannot change volume for %s, volume is fixed", self.name)
+            return
```

Comment:
This check should be unnecessary, `set_volume_level` won't be called for an entity which does not set `MediaPlayerEntityFeature.VOLUME_SET`